blog/_posts/2016-9-19-our-2nd-blog-post.md

```diff
@@ -0,0 +1,24 @@
+---
+title: The Miracle
+layout: post
+created_at: 2016-09-19
+permalink: blog/2016-09-19-The-Miracle
```

Comment:
To make it more consistent, it's best if your filename has the same title as your title/permalink. Can you rename the file to `2016-9-19-The-Miracle.md`?